hazelcast-jet-distribution/src/root/release_notes.txt

```diff
@@ -23,6 +23,42 @@ To expand an issue number into a full GitHub link, use this template:
 
 https://github.com/hazelcast/hazelcast-jet/issues/<issue number>
 
+
+** Hazelcast Jet 4.3 **
+
+Members of the open source community that appear in these release notes:
+
+@guenter-hesse
+@MohamedMandouh
+
+Thank you for your valuable contributions!
+
+
+1. New Features
+
+[pipeline-api] [014] @MohamedMandouh implemented distributed sorting: `BatchStage.sort()` (#2469, #2544)
+[core] [012] Added `JobConfig.suspendOnFailure`: suspend a job on exception instead of cancelling it (#2411)
+
+
+2. Enhancements
+
+[core] @guenter-hesse contributed a test source to benchmark Jet's throughput and latency (#2382)
+[core] [013] Avoid problem where low event rate would stall progress of a processing time-based pipeline (#2485, #2514)
+
+
+3. Fixes
+
+[core] Fixed a bug where a non-keyed aggregating stage would produce no output when no input (#2560, #2567)
+
```

Comment:
Maybe mention the integration of a Jackson-jr bug-fix, which was affecting our JSON-to-POJO mapping, specifically in CDC (https://github.com/FasterXML/jackson-jr/issues/73).

Also, integration of a Debezium fix which solved the problem of losing data on snapshotting in CDC sources for Postgres (https://issues.redhat.com/browse/DBZ-2288).